.clj-kondo/config.edn

```diff
@@ -147,7 +147,6 @@
               :bigquery-cloud-sdk
               :druid
               :druid-jdbc
-              :googleanalytics
```

Comment:
Are you sure?

Reply:
Reverted
https://github.com/metabase/metabase/pull/46980/commits/613995a579c3d533a18c942fcd152d7ca241effb